test/csv/parse/test_liberal_parsing.rb

```diff
@@ -90,4 +90,67 @@ def test_double_quote_outside_quote
                              }),
                  ])
   end
+
+  def test_backslash_quote_double_quote_outside_quote
+    data = %Q{a,""b""}
+    assert_equal([
+                   [["a", "\"\"b\"\""]],
+                   [["a", "\"\"b\"\""]],
+                   [["a", "\"b\""]],
+                 ],
+                 [
+                   CSV.parse(data, liberal_parsing: true),
+                   CSV.parse(data,
+                             liberal_parsing: {
+                               backslash_quote: true
+                             }),
+                   CSV.parse(data,
+                             liberal_parsing: {
+                               backslash_quote: true,
+                               double_quote_outside_quote: true
+                             }),
+                 ])
+  end
+
+  def test_backslash_quote_true_with_unquoted_value
+    data = '\"\"a\"\"'
+    assert_equal([
+                   [["\\\"\\\"a\\\"\\\""]],
+                   [["\"\"a\"\""]],
+                   [["\"\"a\"\""]]
+                 ],
+                 [
+                   CSV.parse(data, liberal_parsing: true),
+                   CSV.parse(data,
+                             liberal_parsing: {
+                               backslash_quote: true
+                             }),
+                   CSV.parse(data,
+                             liberal_parsing: {
+                               backslash_quote: true,
+                               double_quote_outside_quote: true
```

Comment:
Do we need this case?
`double_quote_outside_quote` is meaningless for unquoted value.

Reply:
Remove meaningless test case "unquoted double_quote_outside_quote". cfeb8b58cfe384ea090e78717bb47ea3f168dc24
Also removed from `benchmark/parse_liberal_parsing.yaml`